Spec/RealtimeClientConnection.swift

```diff
@@ -1660,6 +1660,34 @@ class RealtimeClientConnection: QuickSpec {
 
                 }
 
+                // RTN15f
+                it("ACK and NACK responses for published messages can only ever be received on the transport connection on which those messages were sent") {
+                    let options = AblyTests.commonAppSetup()
+                    options.disconnectedRetryTimeout = 0.5
+                    let client = AblyTests.newRealtime(options)
+                    defer { client.close() }
+                    let channel = client.channels.get("test")
+                    var resumed = false
+                    waitUntil(timeout: testTimeout) { done in
+                        client.connection.once(.Connected) { _ in
+                            channel.publish(nil, data: "message") { _ in
```

Comment:
Sorry, I misunderstood the code.  However, as the spec states that the message is resent on the new transport, can we also include that check so that this test code explains what is happening i.e. it's not that on a new connection the ACK is sent, it's actually an ACK that is received from the message being resent.  Do you agree?